src/components/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import { ButtonHTMLAttributes } from 'react';
+import styles from './Button.module.css';
+
+interface PrimaryButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  className?: string;
+}
+
+export default function Button({
+  children,
+  className,
```

Comment:
P1:
classname이 undefined일 수 있고 그러면 `${classname}`일 때 undefined가 들어갈 수 있으니 빈 문자열로 초기화해주세요~

```suggestion
  className = '',
```